src/js/components/CopyToClipboard.js

```diff
@@ -25,21 +25,32 @@ export default class extends React.PureComponent {
         }
     }
 
-    handleCopy = () => {
-        const container = document.createElement('textarea');
+    copyText = async (textToCopy) => {
+        if (navigator.clipboard) {
+            return await navigator.clipboard.writeText(textToCopy);
+        } else {
+            const container = document.createElement('textarea');
+            container.innerHTML = textToCopy;
+            document.body.appendChild(container);
+            container.select();
+            document.execCommand('copy');
+            document.body.removeChild(container);
+            
+        }
+
```

Comment:
```suggestion
        return await navigator.clipboard.writeText(textToCopy);
```

I think it's ok to assume `navigator.clipboard.writeText` is there

compat is +90% https://caniuse.com/mdn-api_clipboard_writetext

Reply:
agree, but then you can just replace `await copyText` with `await navigator.clipboard.writeText(textToCopy);`